VaticanBAV/et/BAVet126.xml

```diff
@@ -0,0 +1,209 @@
+                                <seg part="I">ሰላም፡ ሰላም፡ ለዝክረ፡ ስምኪ፡ ሐዋዝ። </seg>
+                                <gap reason="ellipsis"></gap>
+                                <seg part="F"><supplied reason="omitted" resp="PRS4805Grebaut">ድ</supplied>ንግልናኪ፡ ዘልፈ፡ ይነግር፡ አፉየ</seg>
+                              </q>
+                            </item>
+                            <item xml:id="a2">
+                              <locus from="49v" to="50r"></locus>
+                              <desc type="Subscription"></desc>
+                              <q xml:lang="gez">ተጽሐፈ፡ በምዋ<lb/>ዕለ፡ መንግሥቱ፡<lb/> ለዳግማዊ፡ <persName ref="PRS7006Menilek">ምኒል<lb/>ክ፡</persName> ንጉሠ፡ ነገሥት፡<lb/> ዘኢትዮጵያ፨ ወበ<lb/>መዋዕለ፡
+                              ጳጳስናሁ<lb/><supplied reason="omitted" resp="PRS4805Grebaut">፡</supplied> ለአባ፡ <persName ref="PRS6876Matewos">ማቴዎስ፡</persName> ሊቀ፡<lb/>
+                              ጳጳሳት፡ ዘኢትዮጵያ ወአጽሐፈ<supplied reason="omitted" resp="PRS4805Grebaut">፡</supplied> <del rend="erasure"></del>
+                              ዘውእቱ፡ <supplied reason="omitted" resp="PRS4805Grebaut">ጸ</supplied>ሐፌ፡ ትእዛዝ፡ ወተጽሕፎቱሰ
+                              <date calendar="grace" when="1899" when-custom="1891"
+                                evidence="internal">በ፲፰፻፺፩ ዓመተ፡ ምሕረት፡ ዘመን፡ ሉቃስ፡ ወርኅ፡ ነሓሴ</date>፨
+                              </q>
+                              <q type="translation" xml:lang="en"> Written in the days of the reign of <persName ref="PRS7006Menilek">Mǝnilǝk II</persName>, emperor of Ethiopia and
+                              in the days of the episcopacy of ʾabbā <persName ref="PRS6876Matewos">Mātewos</persName> the archbishop. The ṣaḥafe tәʾzaz <del rend="erasure"></del> ordered the writing
+                              and it took place in the year of mercy 1891 in the era of Luke in the month of Naḥase.</q>
+
+                            </item>
+                          </list>
+                        </additions>
+                        <bindingDesc>
+                          <binding contemporary="true" xml:id="binding">
+                            <decoNote xml:id="b1">Two wooden boards without leather.</decoNote>
+                            <decoNote xml:id="b2" type="bindingMaterial">
+                              <material key="wood">
+                              </material>
+                            </decoNote>
+                          </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                      <origin>
+                        <origDate when="1899">
+                          The <ref target="#a2">subscription</ref> states that the manuscript was written in the year 1899 during the reign
+                          of <persName ref="PRS7006Menilek">Mәnilәk II</persName> when the metropolitan
+                          was <persName ref="PRS6876Matewos">Mātewos</persName>.
+                        </origDate>
```

Comment:
I have slightly changed the encoding here, in accordance with the following example 1: https://betamasaheft.eu/Guidelines/?id=origin